src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SvCigarUtils.java

```diff
@@ -377,4 +377,11 @@ public static int computeAssociatedDistOnRead(final Cigar cigarAlong5To3DirOfRea
 
         return readWalkDist;
     }
+
+    public static int getTotalAlignmentLength(final Cigar cigar ) {
```

Comment:
Maybe is it useful in `CigarUtils` to re-use outside SV code? I do not see specificity with SV here...

Reply:
I agree with @magicDGS, you can call it something like ```countAlignedBases```.

Reply:
done